pkgs/development/r-modules/default.nix

```diff
@@ -950,6 +950,18 @@ let
       preConfigure = "patchShebangs configure";
     });
 
+    littler = old.littler.overrideAttrs (attrs: with pkgs; {
+      buildInputs = [ pcre lzma zlib bzip2 icu which ] ++ attrs.buildInputs;
+      propagatedBuildInputs = [ which ] ++ attrs.propagatedBuildInputs;
+      outputs = [ "out" "man" "doc" ];
+      postInstall = ''
+        install -D inst/bin/r $out/bin/r
```

Comment:
The install process copies dozens of files into `$out`, albeit into strange locations. Now, this code will add even more files, some of them clearly duplicates. I think it would be better to clean up the generated output, i.e. remove files that aren't needed and move others into the right place.

Reply:
Yeah, I'll clean it up. But we should probably decide upon the questions at top.